jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPlugin.kt

```diff
@@ -94,7 +94,6 @@ public class JitsiMeetPlugin() : FlutterPlugin, MethodCallHandler, ActivityAware
 
         pluginBinding = flutterPluginBinding;
 
-        flutterPluginBinding
```

Comment:
I just build the app and I didn't need to change this. Why is this necessary?

Reply:
this looks odd. it shouldn't be ther ein the first place as it is useless that's why i remove that

Reply:
btw what the point of putting it there. it looks like a type in the code and should be removed.

Reply:
You're right, we can remove this.